File: 3617-find-the-original-typed-string-i/find-the-original-typed-string-i.java
class Solution {
    public int possibleStringCount(String word) {
        int output = 1;
        for(int i = 1; i < word.length(); i++){
            char turtle = word.charAt(i-1);
            char hare = word.charAt(i);

            if (turtle == hare)
            output++;
        }
        return output;
        
    }
}

File: 3617-find-the-original-typed-string-i/find-the-original-typed-string-i.js
var possibleStringCount = function(word) {
    output = 1
    for (let i = 1; i < word.length; i++){
        let turtle = word[i-1];
        let hare = word[i];
        if (turtle == hare)
        output++;
    }
    return output;
    
};